### ISSUE TITLE
Window() from transpiler returns different results from ifql

### ISSUE THREAD

**Issue by aanthony1243** [influxdata/platform#97]
ifql query: 
```
from(db:"testdb")
  |> range(start: 2018-05-22T00:00:00Z)
  |> window(every:1s)
  |> map(mergeKey:false, fn: (r) => {_measurement:r._measurement, io_time:r._value,time: r._time})
  |> yield(name:"0")

```

influxql query: 
```
select io_time from testdb..diskio group by time(1s)
```

input data: 
```
#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,long,string,string,string,string
#partition,false,false,false,false,false,false,true,true,true,true
#default,_result,,,,,,,,,
,result,table,_start,_stop,_time,_value,_field,_measurement,host,name
,,1,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:26Z,15204688,io_time,diskio,host.local,disk0
,,1,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:36Z,15204894,io_time,diskio,host.local,disk0
,,1,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:46Z,15205102,io_time,diskio,host.local,disk0
,,1,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:56Z,15205226,io_time,diskio,host.local,disk0
,,1,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:54:06Z,15205499,io_time,diskio,host.local,disk0
,,1,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:54:16Z,15205755,io_time,diskio,host.local,disk0
,,10,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:26Z,648,io_time,diskio,host.local,disk2
,,10,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:36Z,648,io_time,diskio,host.local,disk2
,,10,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:46Z,648,io_time,diskio,host.local,disk2
,,10,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:53:56Z,648,io_time,diskio,host.local,disk2
,,10,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:54:06Z,648,io_time,diskio,host.local,disk2
,,10,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:54:16Z,648,io_time,diskio,host.local,disk2
```

```
result not as expected want(-) got (+):
    		 #datatype,string,long,string,long,dateTime:RFC3339
    		 #partition,false,false,true,false,false
    		 #default,0,,,,
    		 ,result,table,_measurement,io_time,time
    		 ,,0,diskio,15204688,2018-05-22T19:53:26Z
    		-,,0,diskio,648,2018-05-22T19:53:26Z
    		-,,0,diskio,15204894,2018-05-22T19:53:36Z
    		-,,0,diskio,648,2018-05-22T19:53:36Z
    		+,,0,diskio,15204894,2018-05-22T19:53:36Z
    		 ,,0,diskio,15205102,2018-05-22T19:53:46Z
    		-,,0,diskio,648,2018-05-22T19:53:46Z
    		 ,,0,diskio,15205226,2018-05-22T19:53:56Z
    		-,,0,diskio,648,2018-05-22T19:53:56Z
    		 ,,0,diskio,15205499,2018-05-22T19:54:06Z
    		-,,0,diskio,648,2018-05-22T19:54:06Z
    		 ,,0,diskio,15205755,2018-05-22T19:54:16Z
    		+,,0,diskio,648,2018-05-22T19:53:26Z
    		+,,0,diskio,648,2018-05-22T19:53:36Z
    		+,,0,diskio,648,2018-05-22T19:53:46Z
    		+,,0,diskio,648,2018-05-22T19:53:56Z
    		+,,0,diskio,648,2018-05-22T19:54:06Z
    		 ,,0,diskio,648,2018-05-22T19:54:16Z```

**Comment by jsternberg** [comment]
This query is invalid for influxql. Can I get more details on what you mean for this query?

**Comment by jsternberg** [comment]
I think we've resolved the window issues in #127. While this specific query doesn't work, the windowing test in the repository now works so I'm going to close this.
